src/group-configurations/content-groups-section/ContentGroupContainer.jsx

```diff
@@ -0,0 +1,151 @@
+        validationSchema={validationSchema}
+        validateOnChange={false}
+        validateOnBlur={false}
+        onSubmit={onSubmitForm}
+      >
+        {({
+          values, errors, dirty, handleChange, handleSubmit,
+        }) => {
+          const isInvalid = !!errors.newGroupName;
+
+          return (
+            <>
+              <Form.Group
+                className="mt-3 mb-4 content-group-new__input"
+                isInvalid={isInvalid}
+              >
+                <Form.Control
+                  value={values.newGroupName}
+                  name="newGroupName"
+                  onChange={handleChange}
+                  placeholder={formatMessage(messages.newGroupInputPlaceholder)}
+                />
+                {isInvalid && (
+                  <Form.Control.Feedback type="invalid">
+                    {errors.newGroupName}
+                  </Form.Control.Feedback>
+                )}
+              </Form.Group>
+              {isUsedInLocation && (
+                <Alert
+                  variant="warning"
+                  icon={WarningFilledIcon}
+                  className="my-3"
+                >
+                  <p>{formatMessage(messages.alertGroupInUsage)}</p>
+                </Alert>
+              )}
+              <ActionRow>
+                {isEditMode && (
+                  <OverlayTrigger
+                    overlay={(
+                      <Tooltip
+                        id={`delete-restriction-tooltip-${values.newGroupName}`}
+                      >
+                        {formatMessage(
+                          isUsedInLocation
+                            ? messages.deleteRestriction
+                            : messages.deleteButton,
+                        )}
+                      </Tooltip>
+                    )}
+                  >
+                    <Button
+                      disabled={isUsedInLocation}
+                      variant="outline-primary"
+                      onClick={onDeleteClick}
+                    >
+                      {formatMessage(messages.deleteButton)}
+                    </Button>
+                  </OverlayTrigger>
+                )}
+                <ActionRow.Spacer />
+                <Button onClick={onCancelClick} variant="tertiary">
+                  {formatMessage(messages.cancelButton)}
+                </Button>
+                <Button onClick={handleSubmit}>
+                  {formatMessage(
+                    isEditMode ? messages.saveButton : messages.createButton,
+                  )}
+                </Button>
+              </ActionRow>
+              <PromptIfDirty dirty={dirty} />
+            </>
+          );
+        }}
+      </Formik>
+    </div>
+  );
+};
+
+ContentGroupContainer.defaultProps = {
+  groupNames: [],
+  overrideValue: '',
+  isEditMode: false,
+  isUsedInLocation: false,
+  onCreateClick: () => ({}),
+  onDeleteClick: () => ({}),
+  onEditClick: () => ({}),
```

Comment:
[nit]: Let's specify `null` or `undefined` as default values (I looked, this is how it is usually indicated in this project)

Reply:
changed to null